integration-tests/api-gateway/e2e/e2e-happy-path.test.ts

```diff
@@ -133,5 +139,66 @@ describe("End to end happy path journey", () => {
         checkDetails: [{ checkMethod: "data" }],
       },
     ]);
+
+    const startEvents = await pollForTestHarnessEvents(START_EVENT_NAME, sessionId);
+    expect(startEvents).toHaveLength(1);
+    expect(startEvents[0].event).toEqual<AuditEvent>({
+      // clientId is currently unset on start events created by the common session lambda
+      ...baseExpectedEvent(START_EVENT_NAME, undefined, sessionId),
+      extensions: {
+        evidence: [
+          {
+            context: "identity_check",
+          },
+        ],
+      },
+    });
+
+    const reqSentEvents = await pollForTestHarnessEvents(REQUEST_SENT_EVENT_NAME, sessionId);
+    expect(reqSentEvents).toHaveLength(1);
+    expect(reqSentEvents[0].event).toStrictEqual<AuditEvent>({
+      ...baseExpectedEvent(REQUEST_SENT_EVENT_NAME, clientId, sessionId),
+      restricted: {
+        birthDate: [{ value: expect.stringMatching(/^\d{4}-\d{2}-\d{2}$/) }],
+        name: expect.any(Array),
+        socialSecurityRecord: [{ personalNumber: NINO }],
+      },
+    });
+
+    const resReceivedEvents = await pollForTestHarnessEvents(RESPONSE_RECEIVED_EVENT_NAME, sessionId);
+    expect(resReceivedEvents).toHaveLength(1);
+    expect(resReceivedEvents[0].event).toStrictEqual<AuditEvent>({
+      ...baseExpectedEvent(RESPONSE_RECEIVED_EVENT_NAME, clientId, sessionId),
+      extensions: {
+        evidence: { txn: expect.any(String) },
+      },
+    });
+
+    const vcIssuedEvents = await pollForTestHarnessEvents(VC_ISSUED_EVENT_NAME, sessionId);
+    expect(vcIssuedEvents).toHaveLength(1);
+    expect(vcIssuedEvents[0].event).toEqual<AuditEvent>({
+      ...baseExpectedEvent(VC_ISSUED_EVENT_NAME, clientId, sessionId),
+      restricted: {
+        birthDate: [{ value: expect.stringMatching(/^\d{4}-\d{2}-\d{2}$/) }],
+        name: expect.any(Array),
+        socialSecurityRecord: [{ personalNumber: NINO }],
+      },
+      extensions: {
+        evidence: [
+          {
+            checkDetails: [{ checkMethod: expect.any(String) }],
```

Comment:
Same this this - it's quite important which check method was used and I would have thought we'd be able to add that in?